File: src/iniciante/3303/3303.java
import java.io.IOException;
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) throws IOException {
        String palavra;
        Scanner input = new Scanner(System.in);
        
        palavra = input.next();
        
        if (palavra.length() >= 10) {
            System.out.println("palavrao");
        } else {
            System.out.println("palavrinha");
        }
    }
}

File: src/iniciante/3303/3303.cpp
#include <iostream>

using namespace std;
 
int main() {
    string palavra;
    
    cin >> palavra;
        
    if (palavra.length() >= 10) {
        printf("palavrao\n");
    } else {
        printf("palavrinha\n");
    }
 
    return 0;
}